### PR TITLE
Add jxr variants to quality measure

### PR DESCRIPTION
**Pull request by HafnerMichael**

### CODE REVIEW

**Review by Xetiic — APPROVED** [review]
looks good to me

**Comment by m-langer** [comment]
if possible please add this to the quality.py script before importing cv2

```python
### DISCLAIMER ###
# To imort pillow_heif before cv2 is necessary to avoid a bug in the library
# affected devices: macOS on Apple Silicon
import pillow_heif
pillow_heif.register_heif_opener()
### END DISCLAIMER ###
```

File: scripts/quality.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
+import glob
 import os
-import numpy as np
+
```
Comment (m-langer) [inline scripts/quality.py:3]: See: https://github.com/AIS-22/UNI-AIS-MediaDataFormats/pull/79#issuecomment-1821570416

**Review by m-langer — CHANGES_REQUESTED** [review]

**Review by m-langer — APPROVED** [review]
looks good now

**Pull request merged**